data_summaries/AgentInstruct.json

```diff
@@ -0,0 +1,301 @@
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Instruction Following",
+            "Chain-of-Thought"
+        ],
+        "Text Sources": [
+            "database operations"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-4"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "BIRD"
+        ],
+        "Creators": [
+            "Tsinghua University",
+            "The Ohio State University",
+            "UC Berkeley"
+        ],
+        "Licenses": [
+            {
+                "License": "Unspecified",
+                "License URL": ""
+            },
+            {
+                "License": "OpenAI",
+                "License URL": "https://arxiv.org/abs/2310.12823"
+            }
+        ],
+        "License Notes": "The original paper does not provide any information regarding its license. The paper cited for this task https://arxiv.org/pdf/2308.03688.pdf has created the dataset using additional datasets, but it is not clear which of those additional datasets were utilized to create this dataset on databases.",
+        "License Verified By": "Christopher",
+        "Dataset Filter IDs": [
+            "db"
+        ]
+    },
+    "AgentInstruct-mind2web": {
+        "Unique Dataset Identifier": "AgentInstruct-mind2web",
+        "Dataset Name": "mind2web",
+        "Paper Title": "Mind2Web: Towards a Generalist Agent for the Web",
+        "Dataset URL": "https://app.globus.org/file-manager?origin_id=32e6b738-a0b0-47f8-b475-26bf1c5ebf19",
+        "GitHub URL": "https://github.com/OSU-NLP-Group/Mind2Web",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/mind2web",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2306.06070",
+        "Semantic Scholar Corpus ID": 259129428,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Instruction Following",
+            "Chain-of-Thought"
+        ],
+        "Text Sources": [
+            "crowdsourced",
+            "travel websites",
+            "shopping websites",
+            "service websites",
+            "entertainment websites",
+            "information websites"
+        ],
+        "Model Generated": [
+            "OpenAI ChatGPT"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "The Ohio State University"
+        ],
+        "Licenses": [
```

Comment:
I would actually add another entry to each of these: Unspecified and a link to the AgentInstruct repo which has no license.